color/src/dynamic.rs

```diff
@@ -25,6 +26,9 @@ use core::hash::{Hash, Hasher};
 /// When manipulating components directly, setting them nonzero when the
 /// corresponding missing flag is set may yield unexpected results.
 ///
+/// The default value is an opaque white in the [sRGB](Srgb) colorspace with
+/// no missing components. We don't recommend relying upon this default.
+///
```

Comment:
Missing space in "colorspace" and a trailing newline

Reply:
Fixed. (Just in case we do merge it.)

Reply:
Can we add a typos config for this?

Reply:
@DJMcNab Not that I know of ... sure, we can add something to the config, but then it'll warn on `colorspace` as a module name. If you add something to both the identifiers and the words, then you don't get warnings.

Reply:
Maybe that's not a false-positive?

If we're calling it color space everywhere, maybe the module should be `color_space`?